spec/factories/eras.rb

```diff
@@ -4,6 +4,7 @@
     name { Faker::Hipster.sentence }
     start_date { date }
     end_date { Faker::Date.between(from: date, to: Date.today) }
+    week_number { 22 }
```

Comment:
```suggestion
    week_number { Random.rand(0..52)}
```
is there a week 0? Maybe it's 1-52?

Reply:
Love all these additions.  I think that for now I'm find with the `/ 7` method of doing it.  I think a fun refactor would be getting it more precise, however